### DISCUSSION TITLE
keeping track of changing device locations | location hand-over in samedis

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#69] [Ideas]
One potential use case for Samedis: medical devices often go from one department/ward to another. In our hospital they sometimes keep a list of equipment leaving or entering the department, but often they don't . During maintenance we have to search the device and takes time.
Samedis could play a role here perhaps. What if there was a simple (QR-code?) hand-over option in Samedis? A designated peson from a certain department scans the QR code (or enters inventory number manually), then changes the location/hands-over the device to a different designated person/department/location. This will help to keep locations up to date and helps departments to keep track of where their devices are.

**Comment by YvesR** [comment]
@BioMedGPH You can already track this, as this will be part of the change history > Click actions and open device history there to see all changes made to your inventory including location or department changes.

**Comment by BioMedGPH** [comment]
﻿﻿Yes but the idea is users can easily change the location when handing over devices.

**Comment by YvesR** [comment]
@BioMedGPH 
So you're talking about an issue or return of inventory, which could be a central location, or just per department to others, right? So scan via the QR code, then click on handover and select new location?

**Comment by BioMedGPH** [comment]
﻿﻿Yes, like a virtual handshake between two users (maybe mainly read-only) to confirm handover over a device between two locations. To help keep locations up to date so we can actually find devices.

**Reply by YvesR** [reply]
Update: we considering implementing a check-out / check-in functionality where you can hand out items, with timestamp, user, remark, and turn back with timestamp, user, remark (device condition)... this may cover your requirement.
